data-explorer/kusto/concepts/data-purge.md

```diff
@@ -105,22 +105,26 @@ Purge command may be invoked in two ways for differing usage scenarios:
 
 * Human invocation: A two-step process that requires an explicit confirmation as a separate step. First invocation of the command returns a verification token, which should be provided to run the actual purge. This sequence reduces the risk of inadvertently deleting incorrect data.
 
- > [!NOTE]
- > The first step in the two-step invocation requires running a query on the entire data set, to identify records to be purged.
- > This query may time-out or fail on large tables, especially with significant amount of cold cache data. In case of failures,
- > please validate the predicate yourself and after verifying correctness use the single-step purge with the `noregrets` option.
+  > [!NOTE]
+  > The first step in the two-step invocation requires running a query on the entire data set, to identify records to be purged.
+  > This query may time-out or fail on large tables, especially with significant amount of cold cache data. In case of failures,
+  > please validate the predicate yourself and after verifying correctness use the single-step purge with the `noregrets` option.
+
+  If you're issuing the purge manually, first connect to the Data Management service:
+  
```

Comment:
many of the customers use KE, so I think it's Ok to mention both options - we can reference any doc about how to add cluster in KWE (or mention it here), and also mention the `#connect` statement in KE, as in the original doc, but we should mention it only works in KE.

Reply:
Sure thing. I'm going to pull this into the private repo for changes and send to you for review.

Reply:
Agree. Since I don't use Kusto Explorer (it's Windows-only), I didn't know the `#connect` command relates to it, and spent a lot of time trying to make it work on the portal.